policyengine_us/parameters/gov/hhs/fpg.yaml

```diff
@@ -1,6 +1,7 @@
 description: Federal poverty guidelines published by the Department of Health and Human Services.
 first_person:
   CONTIGUOUS_US:
+    1992-01-01: 6_810
```

Comment:
we should add uprating for the of the breakdown values here:
We need metadata for  
```
CONTIGUOUS_US
AK
HI
```

For each `single_person` and `additional_person`